### ISSUE TITLE
Menus and Other Items are Generating Duplicate IDs

### ISSUE THREAD

**Issue by wmcconnell** [ASU-KE/UDS-WordPress-Theme#336]
We have a couple of places where our code is iterating through things and not generating unique HTML IDs:

- The main navigation menu wallker/creator
- The social media icons in the footer
- Probably the footer menu items
- Accordions, as I recall

Anyway, we are getting dinged in things like Lighthouse reports for having duplicate IDs on the page.

**Comment by zainabD** [comment]
@wmcconnell I checked all the elements listed above and I only see the duplicated ID in the main menu walker. (I'm not sure if we need to fix this now since @jkcox  is working on the react version of the walker)

For the other items:
- I confirmed that the accordions are not receiving duplicated IDs. 
- And I cannot see where the social media icons and the footer menu items are receiving duplicated IDs.
Could you please provide more details on how to recreate this issue, or if there is a live website that has all these duplicated IDs.

**Comment by jkcox** [comment]
I don't see any IDs in the react menu - the only thing generated is data attributes: `data-testid="nav-item"`

Otherwise it's only using classes
